boards/arm/stm32l562e_dk/stm32l562e_dk_common.dtsi

```diff
@@ -111,7 +111,7 @@
 &spi1 {
 	pinctrl-0 = <&spi1_sck_pg2 &spi1_miso_pg3 &spi1_mosi_pg4>;
 	pinctrl-names = "default";
-	cs-gpios = <&gpiog 5 (GPIO_ACTIVE_HIGH | GPIO_PULL_UP)>;
+	cs-gpios = <&gpiog 5 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
```

Comment:
Non blocking: A word on this change in commit message wouldn't harm.

Reply:
Done.